collections/Fairfax/MS_Fairfax_16.xml

```diff
@@ -37,49 +36,67 @@
+                           <catchwords>Catchwords at the end of most quires, written in several styles. Up to fol. 110v, the catchwords have no border and occur in the middle of the lower margin. At fol. 118v the catchword is enclosed in a red ink scroll and positioned below the text towards the right as is the one at fol. 126v. Thereafter, some have a red border but most do not.</catchwords></collation>
                      </supportDesc>
+                     <layoutDesc>
+                        <layout columns="1" writtenLines="37 40">Written in long lines, 37–40 per page, <dimensions type="ruled" unit="mm">
+                              <height>155</height>
+                              <width>95</width>
+                           </dimensions>
+                        </layout>
+                     </layoutDesc>
                   </objectDesc>
+                  <handDesc>
+                     <handNote script="cursivaAntiquior">Written in a secretary hand from the middle of the 15th century, the original text seemingly copied by a single scribe, with a separate rubricator and flourisher providing ornaments.</handNote>
+                  </handDesc>
                   <decoDesc>
-                     <decoNote>Fine miniature, border, initial. By <persName key="person_96012549" role="art">William Abell</persName>. Fine drawing (retouched) of 16th century, end, fol. 9. (<ref target="https://catalog.hathitrust.org/Record/000468709">Pächt and Alexander</ref> iii. 974, pl. XCI)</decoNote>
+                     <decoNote type="border">Fine miniature, border, initial. On fol. 15r, a single three-quarter border tied in to a 7-line capital. Attributed to the <persName key="person_2984" role="art">Abingdon Missal Master</persName> (school of <persName key="person_96012549">William Abell</persName>).</decoNote>
+                     <decoNote type="miniature">Fine drawing (retouched) of 16th century, end, fol. 9r. (<ref target="https://catalog.hathitrust.org/Record/000468709">Pächt and Alexander</ref> iii. 974, pl. XCI)</decoNote>
+                     <decoNote type="rubrication">Rubricated in the hand of the scribe. Occasionally, as on fol. 115r, enclosed in a red parallelogram. Marginal headings and running heads in red ink preceded by blue paraphs.</decoNote>
+                     <decoNote type="flourInit">Flourised initials in blue with red penwork extending up and down the margins to begin new items.</decoNote>
+                     <decoNote type="colInit">Red tipping of many upper case letters mainly at the beginning of lines.</decoNote>
                   </decoDesc>
+                  <bindingDesc>
+                     <binding notAfter="1900" notBefore="1800">
+                        <p>Light brown split calf over laminated pulpboard, 19th century; repaired 1949.</p>
```

Comment:
is this still accurate? wasn't it rebound recently? (there may be a conservation report worth checking as well)